packages/graph/hash_graph/lib/graph/src/types/mod.rs

```diff
@@ -1,50 +1,74 @@
-//! Descriptions, and implementation logic, of the Types of elements of the graph.
+//! Model types used across datastores
 
-use sqlx::{types::Uuid, FromRow};
+use std::fmt;
 
-#[derive(Debug, FromRow)]
-pub struct DataType {
-    pub version_id: Uuid,
-    pub schema: serde_json::Value,
-    pub created_by: Uuid,
+use uuid::Uuid;
+
+#[repr(transparent)]
+#[derive(Clone, Copy, Debug, sqlx::Type, PartialEq, Eq)]
```

Comment:
`#[repr(transparent)]` isn't **needed**, no. But because we have `sqlx::transparent` it seems like we should also have it? This is basically just a type alias (with type-safety) for the UUID that lives inside the DB.

Reply:
I see that type safety is the main reason we have this type. But semantically I don't see a reason, why an account id should be copied but borrowed instead. 

I like to **not** block this PR on that.

Reply:
Gotcha, let's follow-up on a call